pkg/tests/observability_install_test.go

```diff
@@ -64,14 +64,13 @@ func installMCO() {
 		instance, err := dynClient.Resource(utils.NewMCOGVR()).Get(MCO_CR_NAME, metav1.GetOptions{})
 		if err == nil {
 			allPodsIsReady = utils.StatusContainsTypeEqualTo(instance, "Ready")
-			return allPodsIsReady
 		}
-		return false
-	}, EventuallyTimeoutMinute*10, EventuallyIntervalSecond*5).Should(BeTrue())
 
-	if !allPodsIsReady {
-		utils.PrintAllMCOPodsStatus(testOptions)
-	}
+		if !allPodsIsReady {
```

Comment:
It can be able to print repeatedly. right?

Reply:
yes, I think it should be the same with 2.3 https://github.com/open-cluster-management/observability-e2e-test/blob/main/pkg/tests/observability_install_test.go#L146  

```
	By("Waiting for MCO ready status")
	Eventually(func() error {
		err = utils.CheckMCOComponentsInHighMode(testOptions)
		if err != nil {
			testFailed = true
			utils.PrintAllMCOPodsStatus(testOptions)
			return err
		}
		testFailed = false
		return nil
	}, EventuallyTimeoutMinute*25, EventuallyIntervalSecond*10).Should(Succeed())
```

Reply:
if we put the `utils.PrintAllMCOPodsStatus(testOptions)` outside of `Eventually`.  the test case will return when `Eventually` failed. we can not get the Pod status.

Reply:
Why cannot we do it in `afterEach`?

Reply:
For the install test case, we do not have `beforeEach` and `afterEach`.  we call install case in `BeforeSuite`.